packages/shared/components/atoms/pills/ActivityTypePill.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
+    import { isParticipationPayload } from '@lib/wallet'
+    import Pill from './Pill.svelte'
+
+    export let activity
```

Comment:
Perfect example for when you actually do need typing. You don't initialize the values, so the type system has no clue what kind of type is expected. Either assign a type or initialize with a known type.

Reply:
This is changed in https://github.com/iotaledger/firefly/pull/3074/files#diff-4d8750ac95e14d34b2ed6dc158586378d054ea9562d43b19226adf663e15b1f1